src/scripts/commands/database.ts

```diff
@@ -0,0 +1,205 @@
+// Commands related to dhis2 database direct manipulation.
+// The generated sql files can be run for example with: d2-docker run-sql <file>
+
+import _ from "lodash";
+import fs from "fs";
+import { command, subcommands, positional, flag, option, optional, string } from "cmd-ts";
+import { HierarchyLevel, OrgUnitPath } from "scripts/common";
+
+export function getCommand() {
+    const removeOrgUnitLevelCommand = command({
+        name: "remove-orgunit-level",
+        description: "Create a sql file that removes organisation units below a hierarchy level",
+        args: {
+            level: positional({
```

Comment:
Usually, named options are more clear, without the `optional` , they will be compulsory.